_posts/2016-09-16-angular-2-animation-important-concepts

```diff
@@ -0,0 +1,236 @@
+---
+layout:     post
+title:      "Angular 2 Animations - Foundation Concepts"
+
+date: 2016-06-08
+imageUrl: '/images/banner/angular-2-component-animations.jpg'
+
+summary: "Animation in Angular 2 is now easy and more intuitive... Learn foundational animation concepts and start animating your Angular 2 components!"
+
+categories:
+  - angular
+
+tags:
+  - angular2
+  - animation
+  - components
+  - relative paths
+
+topic: components
+
+author: thomas_burleson
+---
+
+
+### Our Scenario
+
+Here is a simple Angular 2 component with hide and show features... but no animations (yet):
+
+{% highlight js %}
+{% raw %}
+@Component({
+  selector: 'my-fader',
+    template: `
+    <div *ngIf="visibility == 'shown'" >
+      <ng-content></ng-content>
+      Can you see me? 
+    </div>
+  `
+})
+export class MyComponent implements OnChanges {
+  visibility = 'shown';
+
+  @Input() isVisible : boolean = true;
+
+  ngOnChanges() {
+   this.visibility = this.isVisible ? 'shown' : 'hidden';
+  }
+}
+{% endraw %}
+{% endhighlight %}
+
+This component simply publishes an **@Input** `isVisible` property; which allows other external components to show/hide the text content... without any animations.
+
+<iframe style="width: 100%; height: 600px" src="http://embed.plnkr.co/chNcuU7IoIrN22DF" frameborder="0" allowfullscren="allowfullscren"></iframe>
```

Comment:
Not sure if that was intentionally (and might be fixed later), but this plunk doesn't show the component introduced in this article